### PR TITLE
P0 - Products and Wishlist page

### PR DESCRIPTION
**Pull request by rishimish**
Created Product and Wishlist page

Preview link : https://deploy-preview-3--bee-you.netlify.app/pages/products.html

Screenshot 2022-02-20 at 10 37 20 PM

Screenshot 2022-02-20 at 10 37 40 PM

### CODE REVIEW

File: styles/products.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+main {
+  display: grid;
+  grid-template-columns: repeat(5, minmax(0, 1fr));
+  grid-template-rows: auto;
+  grid-template-areas: "filters items items items items";
+}
+
+.sidebar {
+  grid-area: filters;
+  /* border: 1px solid var(--white); */
```
Comment (ravikumar0403) [inline styles/products.css:10]: can remove commented code if not required

**Review by ravikumar0403 — APPROVED** [review]
left one comment, rest all looks good

File: styles/products.css (hunk 1 above)
Comment (rishimish) [reply styles/products.css:10]: thanks will remove at the end

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+          >
+            <i class="fas fa-shopping-cart m-r-1"></i>Add to Cart
+          </button>
+          <button class="primary-btn btn m-1 vertical-center">Buy Now</button>
+        </div>
+        <div class="card vertical-card">
+          <div>
+            <img
+              src="https://assets.myntassets.com/h_720,q_90,w_540/v1/assets/images/14852062/2021/8/10/c43e2ced-edcf-4337-84eb-9961035656201628592699342-Levis-Men-Jeans-3711628592698640-1.jpg"
+              alt=""
```
Comment (bharati-21) [inline pages/wishlist.html:112]: You can include any value for alt tag/

**Review by bharati-21 — APPROVED** [review]
You can use a formatter to format the code.

Looks good to be merged.

**Comment by rishimish** [comment]
> You can use a formatter to format the code.
> 
> Looks good to be merged.

Using prettier on save but I guess it is not working how it's expected 😅
thanks for the review

**Pull request merged**